pkg/gui/controllers/tags_controller.go

```diff
@@ -236,6 +243,10 @@ func (self *TagsController) create() error {
 	})
 }
 
+func (self *TagsController) merge(tag *models.Tag) error {
+	return self.c.Helpers().MergeAndRebase.MergeRefIntoCheckedOutBranch(tag.RefName())
```

Comment:
We should focus the local branches view after merging so that the user has more context on what's going on. Let me know if you need help with that

Reply:
Added this in f6a24235ac4e, let me know if it's preferred to squash down these commits.